docs/control-structures.rst

```diff
@@ -598,6 +598,7 @@ function calls which will cause a Panic.
 A Panic exception is generated in the following situations.
 The error code supplied with the error data indicates the kind of panic.
 
+#. 0x00: Failed to set up the "``delegatecall``"-check during library deployment.
```

Comment:
Could also just say:
```suggestion
#. 0x00: Internal errors.
```

Reply:
Actually `libsolutil/ErrorCodes.h` just calls this "Generic panic" and `0x01` as "Generic assert / user-inserted assert".

Reply:
`0x00` is the generic "any other reason" panic code, so please change.

Reply:
Fixed.